en_ewt-ud-train.conllu

```diff
@@ -179780,9 +179782,9 @@
 # text = And it's not all about how much you earn, cost of living is much higher in many cities in Australia.
 1	And	and	CCONJ	CC	_	10	cc	10:cc	_
 2-3	it's	_	_	_	_	_	_	_	_
-2	it	it	PRON	PRP	Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs	10	nsubj	10:nsubj	ToCheck=Subject
+2	it	it	PRON	PRP	Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs	3	nsubj	3:nsubj	_
 3	's	be	AUX	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	10	cop	10:cop	_
-4	not	not	PART	RB	_	10	advmod	10:advmod	_
+4	not	not	PART	RB	_	3	advmod	3:advmod	_
```

Comment:
The original was correct for this sentence—copulas attaching as `cop` should not have dependents.